presto-docs/src/main/sphinx/release/release-0.248.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.248
+=============
+
+**Highlights**
+==============
+* New aggregation function MAP_UNION_SUM.
+* New scalar function DESTRUCTURE_TDIGEST.
+* Support for session property managers, that can override session properties set by a user.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve output buffer memory tracking to reduce lock contention.
+* Add support for overriding session properties using session property managers :doc:`/admin/session-property-managers`. Setting `overrideSessionProperties` to true will cause the property to be overridden and remain overridden even if subsequent rules match the property but don't have `overrideSessionProperties` set.
+* Add support to drop multiple UDFs at the same time.
+* Add a REST endpoint to get TaskInfo, without directly going to worker endpoint.
+* Add new aggregation function :func:`map_union_sum`.
+* Add new scalar function DESTRUCTURE_TDIGEST to expose the internal TDigest data structure as a ROW
```

Comment:
Is this documented anywhere?  if so, it should have a link to the function documentation like the other function release notes.  If not, we should create a task for whoever added it to add documentation.

Reply:
There's no documentation for any TDigest function at all, not just this one

Reply:
I found this issue https://github.com/prestodb/presto/issues/15674, so looks like it's being tracked